patches/MAGETWO-99902__pass_store_view_scope_in async_web_api__2.3.2.patch

```diff
@@ -0,0 +1,948 @@
+diff --git a/vendor/magento/module-amqp/Plugin/AsynchronousOperations/MassConsumerEnvelopeCallback.php b/vendor/magento/module-amqp/Plugin/AsynchronousOperations/MassConsumerEnvelopeCallback.php
```

Comment:
Change first line to be same as our other patches with parameters. Also drop index